web/vuenuxt/nuxt_ssr/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:lts
+COPY service/ /app/
```

Comment:
can you move workdir up so it's before copy. also can you split copy into two where the first copy copies package*.json and the second one happens after `RUN npm install`